site/public/js/ta-grading-rubric.js

```diff
@@ -3414,6 +3419,7 @@ function injectInstructorEditComponentHeader(component, showMarkList) {
  */
 function injectGradingComponent(component, graded_component, editable, showMarkList) {
     const student_grader = $('#student-grader').attr('is-student-grader');
+    console.log('rendering:', graded_component);
```

Comment:
Is this left over from your local testing?

Reply:
yeah oops